easybuild/easyconfigs/l/libxc/libxc-6.1.0-GCC-12.2.0.eb

```
@@ -18,7 +18,12 @@ builddependencies = [
     ('Perl', '5.36.0'),
 ]
 
-local_common_configopts = "-DENABLE_FORTRAN=ON -DENABLE_FORTRAN03=ON -DENABLE_XHOST=OFF"
+local_common_configopts = "-DENABLE_FORTRAN=ON -DENABLE_FORTRAN03=ON -DENABLE_XHOST=OFF "
```

```suggestion
local_common_configopts = "-DENABLE_FORTRAN=ON -DENABLE_XHOST=OFF "
```

`ENABLE_FORTRAN03` is no longer used:

```
CMake Warning:
  Manually-specified variables were not used by the project:

    ENABLE_FORTRAN03
```

See https://gitlab.com/libxc/libxc/-/merge_requests/258/diffs